README.rst

```diff
@@ -87,7 +87,7 @@ To see what coala can do for your language, run:
 +----------------------------+----------------------------+----------------------------+
 | Go                         | Python 3                   | YAML                       |
 +----------------------------+----------------------------+----------------------------+
-| Haskell                    | R                          |                            |
+| Haskell                    | R                          | HAML                       |
```

Comment:
Doesn't keep the alphabetical order ;)
(I know this layout is anyway crappy to maintain... I think we got an issue to do it differently.)

Reply:
Do i have to rearrange it ?